documentation/src/test/java/snippets/GettingStarted.java

```diff
@@ -8,7 +8,7 @@
 
     public static void main(String[] args) {
         Multi.createFrom().items("hello", "world")
-                .onItem().apply(s -> s.toUpperCase() + " ")
+                .onItem().transform(s -> s.toUpperCase() + " ")
```

Comment:
The documentation should also be updated.

Reply:
Weird, it's been changed on my machine!

Reply:
Ah you mean the text?

Reply:
Yes.

Reply:
Found 1 reference, updated.